### DISCUSSION TITLE
[RFC] Allow ||| operator or similar when using {% setcontent %} on multiple fields

### DISCUSSION THREAD

**Discussion by nestordedios** [bolt/core#1696] [Ideas / Feature requests]
Hi, 

I open this as a RFC since I do not know if this will be supported in Bolt 4 or done differently than in Bolt 3.

In Bolt 3 documentation https://docs.bolt.cm/3.7/templating/content-fetching#and-and-or you can read:

> By using the ||| -operator (three pipes) you can create an OR-part for multiple columns. For example:
> 
> {# Select users from Amsterdam that match either username 'pete' or firstname 'Mike' #}
> {% setcontent mypages = 'pages' where { city: 'Amsterdam', 'username ||| firstname': 'pete ||| Mike' } %}
> 
> {# Query output:
>     WHERE ( (city = 'Amsterdam') AND ( (username = 'pete') OR (firstname = 'Mike') ) )
> #}

That seems not to work on Bolt 4. It would be nice to be able to set content using the OR operator for multiple fields.

**Comment by bobdenotter** [comment]
My vote would be to burn this with fire. 🔥 🔥 

This was also one of those things that were _both_ incredibly hard to implement properly and it was often very confusing for the end user. 

In practice, i think we hardly ever need this.
